.github/workflows/ci.yml

```diff
@@ -102,7 +102,7 @@ jobs:
           PINECONE_API_KEY: ${{ secrets.PINECONE_API_KEY }}
           PINECONE_ENVIRONMENT: ${{ secrets.PINECONE_ENVIRONMENT }}
           TRACELOOP_API_KEY: ${{ secrets.TRACELOOP_API_KEY }}
-        run: npx nx affected -t test --exclude='sample-app' --parallel=3
```

Comment:
:question:  I'm curious, why did you decide to remove the parallelization?

Reply:
@paolorechia There's some issue with tests flakiness which I suspect to be related to parallelism, actually not this kind though but multithreading, so I gave it a try a hopeless try :)